renderer/plugin.js

```diff
@@ -48,10 +53,12 @@ org.ekstep.questionunitmcq.RendererPlugin = org.ekstep.contentrenderer.questionU
     MCQController.renderQuestion(); // eslint-disable-line no-undef
     if (this._question.state && _.has(this._question.state, 'val')) {
       this._selectedIndex = this._question.state.val;
-      $("input[name='radio']", $(this._constant.mcqParentDiv))[this._selectedIndex].checked = true; // eslint-disable-line no-undef
-      if (this._question.config.layout == "Horizontal") {
-        $($("input[name='radio']")[this._selectedIndex]).parents('.option').addClass('selected');
-      }
+      var selectedIndex = this._selectedIndex;
+      _.each($(".org-ekstep-questionunit-mcq-option-element"), function(optionElement, index){
+        if(index == selectedIndex){
+          $(optionElement).trigger("click");
```

Comment:
Why are you triggering click event, It will log telemetry interact event. Only apply required styles by updating the relevant obj/key.